### ISSUE TITLE
Should the Unfettered be able to be bribed...

### ISSUE THREAD

**Issue by Amazinite** [endless-sky/endless-sky#2308]
when you are assisting the Wanderers against them in the invasion? At that point, it seems like the Unfettered wouldn't take your bribes since you're hurting their invasion effort and helping the Wanderers.

**Comment by Tadrix** [comment]
On the other hand, by bribing them you are directly financing their war effort…

**Comment by Amazinite** [comment]
Maybe, but they really don't seem to have much use for credits. Where would they be buying things from to advance their war effort?

**Comment by MessyMix** [comment]
They'd make them, I would think. It takes money to manufacture weaponry, even internally. There are people to feed and machines to build, and that'll all take money. Not to mention extracting raw materials from the lush worlds that the Wanderers left behind.

**Comment by Tadrix** [comment]
It alone would cause inflation, but consider that people who bribe them also come to trade, and then the Unfettered get to pay for these goods with the money you just bribed them…

**Comment by DingusShingleton** [comment]
Looked at the other way the hai get to sell goods to the person who just paid to be there

**Comment by Amazinite** [comment]
I think it's important to note that their planets can't be bribed after the invasion, but they ships can.

**Comment by Tadrix** [comment]
Did it change at some point in development (and why)? Because I remember bribing planets to land on them in 0.9.4, after starting to help Wanderers.

**Comment by Bladewood** [comment]
You can land on their planets at first after bribing them, but once the Unfettered conquer a world in Wanderer space, they won't let you land anymore.

**Comment by Tadrix** [comment]
Guess they grow more confident…

However, their ships accepting bribes is still normal. You are financing them, after all, and their war of conquest is against Wanderers, not you.

**Comment by Amazinite** [comment]
But if you're using the "you're funding their war effort" argument then why wouldn't they take your money to let you land as well?

**Comment by Tadrix** [comment]
By the way, why they aren't taking in the first place? Wasn't it to prevent player improving relations with Unfettered during the war?

**Comment by gunqqer** [comment]
I think that is why we can't (gameplay wise). Lore wise there is no real reason but having the player do missions for the unfettered (I think you can to food tributes at this point) and gain rep during a war with them doesn't make logicyness. But then that is inconsistent with when you are at war with the republic... can of worms gunqqer, dont open it...

**Comment by geojak** [comment]
I think it wold be better if the player can freely bribe to land at any stage. More power to the player in a sandbox game is a good think.

**Comment by monticore162** [comment]
the unfettered are short sighted and will jump at any opportunity to make quick gains so I say yes they should be bribable.

**Comment by Hurleveur** [comment]
I agree they should be bribable at all time, except if you prove to have very little honor in some way.

The idea is you have to pay a fee for safe passage, and indeed their conflict is not with the Wanderers. And then gameplay wise its nice to be able to bribe them especially since the battles were made more difficult recently.

**Comment by Hurleveur** [comment]
What would make sense is to increase the amount needed to bribe during the war, say by 2x.

**Comment by Hurleveur** [comment]
> I think it's important to note that their planets can't be bribed after the invasion, but their ships can.

Isnt it only the new planets? I think they still allow bribes for hai space worlds - but they probably dont want a spy on the new ones?

**Comment by TheGiraffe3** [comment]
Would the bribes be changed back to normal during the UHai/Wanderer peace period or not?
As far as I can tell, all we need to do is add
```
	government "Hai (Unfettered)"
		"bribe" 0.04
```
to `event: Sich'ka'ara empty` and that should fix the issue.
